store/src/java/com/zimbra/cs/service/formatter/MobileConfigFormatter.java

```diff
@@ -0,0 +1,1068 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2016 Synacor, Inc.
```

Comment:
Do we need to update the dates in the license block?

Reply:
done